core/lib/dal/migrations/20241209150000_create_via_votes.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS via_votable_transactions (
+    l1_batch_number BIGINT NOT NULL REFERENCES l1_batches (number) ON DELETE CASCADE,
```

Comment:
When running this layer on the verifier node, this FK  reference will be an issue as we will not have a l1_batches table in the verifier DB. I suggest to remove this FK reference

Reply:
let me do that in next PR with other changes for verifier